ts/packages/ts/src/commands/UpsertCommand.ts

```diff
@@ -72,6 +72,8 @@ export class UpsertCommand extends Command<void> {
       })
       .value();
 
+    await store.init();
```

Comment:
this should be called at the beginning of the command

Reply:
Done